ui/pages/groups/[id].js

```diff
@@ -95,18 +95,20 @@ export default function GroupDetails() {
   const router = useRouter()
   const id = router.query.id
   const page = Math.max(parseInt(router.query.p) || 1, 1)
-  const limit = 999
-  const { isAdmin } = useUser()
-  const { data: group, mutate: mutate } = useSWR(`/api/groups/${id}`)
+  const limit = 10
+  const { user, isAdmin } = useUser()
+  const { data: group, mutate } = useSWR(`/api/groups/${id}`)
   const {
     data: { items: users, totalCount, totalPages } = {},
     mutate: mutateUsers,
   } = useSWR(`/api/users?group=${group?.id}&limit=${limit}&p=${page}`)
-  const { data: { items: allUsers } = {} } = useSWR(`/api/users?limit=999`)
+  const { data: { items: allUsers } = {} } = useSWR(`/api/users?limit=1000`)
```

Comment:
should the match the limit declared on line 98?
```suggestion
  const { data: { items: allUsers } = {} } = useSWR(`/api/users?limit={limit}`)
```

Reply:
no, we want to get all the users so that can be listed in the dropdown options. There should have a better way to do it, but we don't want to paginate the users for sure